src/app/cheat-sheets/game-base/train-colors/train-colors.data.ts

```diff
@@ -14,35 +14,42 @@ export const TRAIN_COLOR_DATA: RawData<TrainColorsData> = {
       {
         icon: FactorioIcons.Icons_IronOre,
         rgb: [0, 140, 255],
+        type: 'Iron ore',
```

Comment:
Am I adding the `type:` field data in the right file? Is the file I'm editing in this PR the source of truth?

I just found out you have a [`download-icons-enum.ts`](https://github.com/deniszholob/factorio-cheat-sheet/blob/master/scripts/download-icons-enum.ts) script that pulls in _something_ from the [`deniszholob/icons-factorio` repository](https://github.com/deniszholob/icons-factorio).

Reply:
For "type" looks like an override to the icon name using a replace function that used to work for the wiki icon naming format)
          {{ item.type ? item.type : (item.icon | replace) }}
          https://github.com/deniszholob/factorio-cheat-sheet/blob/master/src/app/pipes/replace.pipe.ts#L5
          
I think instead we can just change from

```ts
{{ item.type ? item.type : (item.icon | replace) }}
```

to
```
{{ item.type  ?  item.type  :  FactorioIcons[item.icon].display }}
```